### PR TITLE
Jonpan lab05 pick6

### PR DESCRIPTION
**Pull request by notvert**

### CODE REVIEW

**Review by brucestull — CHANGES_REQUESTED** [review]
[Lab 5: Pick6](https://github.com/PdxCodeGuild/class_HB2/blob/main/1%20Python/labs/05%20Pick6)

**Needs Work**

Comments:
* Labs need to be on their own branch. See me on how to break out labs 3 and 5 so they aren't on this same branch.

**Comment by notvert** [comment]
Removed Lab03 from this branch

File: code/jonpan/lab05.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    return matches
+
+def pick_number(x=6):
+    return random.sample(range(1, 99), x)
+
+balance = 0
+loops = 100000
+ticket_cost = (loops * -2)
+
+for x in range(loops):
```
Comment (brucestull) [inline code/jonpan/lab05.py:17]: * `winning_ticket` should be determined outside the loop. We want to compare 100000 player tickets to one `winning_ticket`.

File: code/jonpan/lab05.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    elif match_counter == 4:
+        balance += 50000
+    elif match_counter == 5:
+        balance += 1000000
+    elif match_counter == 6:
+        balance += 25000000
+
+payoff = balance + ticket_cost
+
+print(payoff)
```
Comment (brucestull) [inline code/jonpan/lab05.py:37]: * Lab needs to provide `ROI`.

**Review by brucestull — CHANGES_REQUESTED** [review]
[Pick6](https://github.com/PdxCodeGuild/class_HB2/blob/main/1%20Python/labs/05%20Pick6.md)

**Needs Work**

**Review by brucestull — CHANGES_REQUESTED** [review]
**Needs Work**

Comments:
* Need to resolve merge conflict.
* I think that students have been deleting old lab 'files' from their branch since I haven't been doing an adequate job of explaining the concept of what git is for. Old labs 'files' should show up on new branches, BUT they only become noticable when we try to edit them. It's expected for old labs which have been merged into main, and then become part of new lab branches, to show on new branches. We don't want to delete them. They are part of branch `main` now.
* It can be quite a challenge to understand these concepts. Let's discuss and also try to find some git tutorials to it makes sense.

Please see me so we can discuss. This is an important concept.

**Comment by notvert** [comment]
I'm going to create a new branch to resolve this error like I did for previous labs.

**Pull request closed without merging**